test/freenet/support/HTMLNodeTest.java

```diff
@@ -533,12 +533,12 @@ private String readFirstLine(String aString) {
 	public void testGenerate_fromHTMLNode_textareaDivA() {
 		HTMLNode methodHTMLNode;
 		String[] nodeNamesArray = {"textarea","div","a"};
-		for(int i=0;i<nodeNamesArray.length;i++) {
-			boolean newlines = new HTMLNode("a").newlineOpen(nodeNamesArray[i]);
-			methodHTMLNode = new HTMLNode(nodeNamesArray[i],
-					SAMPLE_OKAY_ATTRIBUTE_NAME,SAMPLE_ATTRIBUTE_VALUE);
-			assertEquals(generateFullNodeOutput(nodeNamesArray[i], 
-					SAMPLE_OKAY_ATTRIBUTE_NAME,SAMPLE_ATTRIBUTE_VALUE,"", newlines),
+		for (String s : nodeNamesArray) {
```

Comment:
Can you pick a better name for this variable than `s`, please? `nodeName` might be appropriate.